src/plugins/analysis/architecture_detection/code/architecture_detection.py

```diff
@@ -59,21 +59,26 @@ class MetaDataDetector:
     '''
 
     architectures = {
+        'ARC': ['ARC Cores'],
         'ARM': ['ARM'],
+        'AVR': ['Atmel AVR'],
         'PPC': ['PowerPC', 'PPC'],
         'MIPS': ['MIPS'],
         'x86': ['x86', '80386', '80486'],
         'SPARC': ['SPARC'],
+        'RISC-V': ['RISC-V'],
         'RISC': ['RISC', 'RS6000', '80960', '80860'],
         'S/390': ['IBM S/390'],
         'SuperH': ['Renesas SH'],
+        'ESP': ['Tensilica Xtensa'],
         'Alpha': ['Alpha'],
         'M68K': ['m68k', '68020']
     }
     bitness = {
+        '8-bit': ['8-bit'],
         '16-bit': ['16-bit'],
         '32-bit': ['32-bit', 'PE32'],
-        '64-bit': ['64-bit', 'aarch64', 'x86-64', '80860']
+        '64-bit': ['64-bit', 'aarch64', 'x86-64', 'MIPS32', 'MIPS64', '80860']
```

Comment:
MIPS32 is 32-bit :-)

Reply:
whoops good catch :sweat_smile: -> moved to '32-bit'